rir/src/interpreter/interp.cpp

```diff
@@ -2569,15 +2569,13 @@ SEXP evalRirCode(Code* c, Context* ctx, SEXP* env,
 
         INSTRUCTION(set_shared_) {
             SEXP val = ostack_top(ctx);
-            if (NAMED(val) < 2) {
-                SET_NAMED(val, 2);
-            }
+            ENSURE_NAMEDMAX(val);
             NEXT();
         }
 
         INSTRUCTION(make_unique_) {
             SEXP val = ostack_top(ctx);
-            if (NAMED(val) == 2) {
+            if (NAMED(val) > 2) {
```

Comment:
@o- I know it's already merged, but why is this `>` and not `>=`?

Reply:
uh, good catch. thanks